apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/components/template-selector-control.js

```diff
@@ -32,29 +32,33 @@ function TemplateSelectorControl( {
 			help={ help }
 			className={ classnames( className, 'template-selector-control' ) }
 		>
-			{ templates.map( ( option, index ) => (
-				<div key={ `${ id }-${ index }` } className="template-selector-control__option">
-					<button
-						type="button"
-						id={ `${ id }-${ index }` }
-						className="template-selector-control__label"
-						value={ option.value }
-						onClick={ handleButtonClick }
-						aria-describedby={ help ? `${ id }__help` : undefined }
-					>
-						<div className="template-selector-control__media-wrap">
-							{ option.preview && (
-								<img
-									className="template-selector-control__media"
-									src={ option.preview }
-									alt={ 'Preview of ' + option.label }
-								/>
-							) }
-						</div>
-						{ option.label }
-					</button>
-				</div>
-			) ) }
+			<ul className="template-selector-control__options">
+				{ templates.map( ( option, index ) => {
+					return (
```

Comment:
What's the benefit of the additional return rather than `() => ()`?

Reply:
Accidentally left this in when I was debugging.